Editors/Audio/Presentation/AudioEditor/AudioEditorViewModelHelpers.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using Editors.Audio.Presentation.AudioEditor.ViewModels;
+using Editors.Audio.Storage;
+using static Editors.Audio.Presentation.AudioEditor.DynamicDataGrid;
+using static Editors.Audio.Presentation.AudioEditor.ViewModels.AudioEditorViewModel;
+
+namespace Editors.Audio.Presentation.AudioEditor
+{
+    public class AudioEditorViewModelHelpers
+    {
+        public static Dictionary<string, List<string>> DialogueEventsWithStateGroupsWithQualifiers { get; set; } = [];
```

Comment:
Why is this static? That makes it shared across all instances. Should the whole class be static?